### PR TITLE
Accessibility styles + responsive mixin

### PR DESCRIPTION
**Pull request by arh30**
- Adds override of focus styles. I've gone with just overriding the outline for now, using box-shadow only where needed! Lets keep outline always, I've previously used JS to detect if user is tabbing but i've realised that's a flawed method.
- Adds responsive mixins
- Changed primary and secondary colour to make the opening site more welcoming :D 
- Couple more tweaks like adding padding to the nav links, to make for a prettier focus.

Where we're at:
Screenshot 2019-06-03 at 15 26 16
Screenshot 2019-06-03 at 15 25 45

### CODE REVIEW

**Comment by ikonitas** [comment]
build is failing 🤷‍♂️

File: {{cookiecutter.project_slug}}/static/src/js/base.js [hunk 1]
```diff
@@ -1,6 +1,6 @@
 const main = document.querySelector('.main');
 
-// Stretch main to fill height of screen minus header and footer
+// Stretch main height to fill the screen
 if (main) {
     const footer_height = document.querySelector('.footer').offsetHeight;
     const header_height = document.querySelector('.header').offsetHeight;
```
Comment (martincoote) [inline {{cookiecutter.project_slug}}/static/src/js/base.js]: Might we be better off doing this with flexbox or css grid? if we set body to `min-height: 100vh;` `display: flex;` `flex-dir: column;` and set `<main>` to have `flex-grow: 1;` we can avoid the need for JS.

**Comment by martincoote** [comment]
Looking good apart from the above comment.

File: {{cookiecutter.project_slug}}/static/src/js/base.js (hunk 1 above)
Comment (arh30) [reply {{cookiecutter.project_slug}}/static/src/js/base.js]: possibly, yes! Make a PR for that when you get a chance and I'll review. This is just what we've used in the past and works well, it's not the worst thing to lose if JS is playing up, but if there's a nice sass solution lets do that!

Comment (martincoote) [reply {{cookiecutter.project_slug}}/static/src/js/base.js]: Have added a card linking to the comment in to the frontend improvements backlog. https://favro.com/organization/b03d6d6d9b11b61167ac4246/418c53a770389640d415a230?card=Dev-12017

**Pull request merged**